src/sync-endpoint-cloud-setup.rst

```diff
@@ -74,60 +74,93 @@ Setting up a Droplet
+Enabling a firewall to prevent unintended traffic
+"""""""""""""""""""""""""""""""""""""""""""""""""""
 
 1. On the resources tab of the main DigitalOcean page, click on the :guilabel:`Droplet` you created.
 
-  .. image:: /img/setup-digital-ocean/do-droplets.png
+  .. image:: /img/setup-digital-ocean/syncendpoint-droplet.png
+   :width: 600
+
+2. On the droplet page, click on :guilabel:`Networking`, this will take you to the droplet's networking page.
+  .. image:: /img/setup-digital-ocean/do-networking.png
    :width: 600
 
-2. Now, click on the :guilabel:`Console` link in the upper-right corner of the page
+3. Next, scroll down to the :guilabel:`Firewalls` section and click on the :guilabel:`Edit` button.
+
+  .. image:: /img/setup-digital-ocean/do-edit-firewall.png
+   :width: 600
+
+4. On the :guilabel:`Firewalls` tab of the :guilabel:`Networking` page,  and click on the :guilabel:`Create Firewall` button.
+
+  .. image:: /img/setup-digital-ocean/do-firewall.png
+   :width: 600
+
+5. Set a name for your firewall and modify the inbound rules to match the inbound rules specified in the picture below (SSH, HTTP, HTTPS and port (40000) for admin interface). Attach the firewall to the desired droplet. Leave the outbound rules as-is.
+
+  .. image:: /img/setup-digital-ocean/do10.png
+   :width: 600
+
+  .. image:: /img/setup-digital-ocean/do11.png
+   :width: 600
+
+6. After going through the instructions for :ref:`Creating a Sample User <sync-endpoint-setup-create-user>`, we no longer need access to this admin interface anymore. This admin interface is running on port 40000, and in order to ensure that this admin interface is not publicly accessible to anyone, we want to remove the rule that accepts incoming traffic to that port. Go ahead and remove the following rule:
+
+  .. image:: /img/setup-digital-ocean/do12.png
+   :width: 600
+
+.. _sync-endpoint-setup-digital-ocean-connecting:
+
+Connecting to your Droplet
+"""""""""""""""""""""""""""""
+
+1. On the droplet page, click on the :guilabel:`Console` link in the upper-right corner of the page
 
   .. image:: /img/setup-digital-ocean/do-console.png
    :width: 600
 
-3. A console window will now open up. If you chose the **password** authentication, you will be asked to enter your username and then asked for a password.
+2. A console window will now open up. If you chose the **password** authentication, you will be asked to enter your username and then asked for a password.
 
   .. image:: /img/setup-digital-ocean/do-console-terminal.png
    :width: 600
 
-4. Before running our launch scripts, we need to check our logs to ensure that all the packages have been successfully installed, which should take about 2-3 minutes. The droplet may also reboot in this time.
+3. Before running our launch scripts, we need to check our logs to ensure that all the packages have been successfully installed, which should take about 2-3 minutes. The droplet may also reboot in this time.
```

Comment:
Just for consistency of style, "we need to check our logs" should be "you need to check your logs"